plugins/modules/ec2_instance.py

```diff
@@ -948,6 +952,11 @@
                     returned: always
                     type: str
                     sample: default
+        additional_info:
+            description: Reserved for Amazon's internal use. 
```

Comment:
```suggestion
            description: Reserved for Amazon's internal use.
```

https://github.com/ansible-collections/amazon.aws/actions/runs/6798846893/job/18485232245?pr=1828

```
ERROR: Found 2 pep8 issue(s) which need to be resolved:
ERROR: plugins/modules/ec2_instance.py:378:44: W291: trailing whitespace
ERROR: plugins/modules/ec2_instance.py:956:61: W291: trailing whitespace
```

Reply:
Updated in latest rev.